src/System.Data.Common/tests/System/Data/DataRowTest2.cs

```diff
@@ -707,6 +700,7 @@ public void GetParentRows_ByNameDataRowVersion()
             */
         }
 
+        //TODO: Unused?
         private void CheckRowVersion(DataRow dr)
```

Comment:
This method seems to be unused.

Reply:
If it's not used, feel free to delete it.